docs(tem): quickstart and how-tos (#942)
scaleway/docs-content: managed-services/transactional-email/quickstart.mdx

---
meta:
  title: Transactional Email - Quickstart
  description: This page shows you how to get started with Scaleway's Transactional Email service.
content:
  h1: Transactional Email - Quickstart
  paragraph: This page shows you how to get started with Scaleway's Transactional Email service.
dates:
  validation: 2022-11-07
  posted: 2022-11-07
---

Transactional Email is a platform that allows Scaleway clients to send [transactional emails](managed-services/transactional-email/concepts/#transactional-email) with high quality [deliverability](/managed-services/transactional-email/concepts/#deliverability).

> In this Quickstart, we show you how to configure your domain with Transactional Email, how to add SPF, DKIM and MX records to your domain, and how to delete your domain.

Reviewer: ```suggestion
In this quickstart, we show you how to configure your domain with Transactional Email, how to add SPF, DKIM, and MX records to your domain, and how to delete your domain.
```

<Message type="requirement">

>   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)

Reviewer: ```suggestion
  - You have an account and are logged into the [Scaleway console.](https://console.scaleway.com)
```

>   - You have a domain name

Reviewer: ```suggestion
  - You have a domain name.
```

</Message>

## How to configure your domain with Transactional Email

> 1. Click **Transactional Email** in the Network section of the [console](https://console.scaleway.com/) side menu. The **Transactional Email** overview page displays.

Reviewer: ```suggestion
1. Click **Transactional Email** in the Managed Services section of the [console](https://console.scaleway.com/) side menu. The **Transactional Email** overview page displays.
```

2. Click **Add a Domain**.
3. Enter the domain name you want to use with Transactional Email.
4. Click **Validate domain name** to confirm.

<Message type="important">
 You need to configure your [SPF](/managed-services/transactional-email/concepts/#spf-record), [DKIM](/managed-services/transactional-email/concepts/#dkim-record) and [MX](/managed-services/transactional-email/concepts/#mx-record) records to verify your domain and to be able to send emails.
</Message>

## How to add SPF, DKIM and MX records to your domain

After you have configured your domain with Transactional Email, you are prompted to verify your domain by adding SPF, DKIM and MX records. You can choose to do so right after you have validated your domain, or at a later time.

> 1. Click **Transactional Email** in the Network section of the [console](https://console.scaleway.com/) side menu. Your newly-added domain displays.

Reviewer: ```suggestion
1. Click **Transactional Email** in the Managed Services section of the [console](https://console.scaleway.com/) side menu. Your newly-added domain displays.
```

2. Click your domain. You are redirected to the SPF, DKIM and MX configuration page.
3. Copy the value that applies to your case to add an SPF record.
    <Lightbox src="scaleway_spf_record_different_values.webp" alt="" />
4. Add a `TXT` type record in the root zone of your domain on your DNS zone.
5. Paste the value you have copied in the root zone of your domain.
    <Lightbox src="scaleway_spf_record.webp" alt="" />
6. Add an additional `TXT` type record in the root zone of your domain on your DNS zone.
7. Copy the name that displays in the console.
    <Lightbox src="scaleway_dkim_record_name.webp" alt="" />
8. Paste the name you have copied in the root zone of your domain on your DNS zone.
9. Copy the value that displays in the console and paste it the root zone of your domain on your DNS zone.
    <Lightbox src="scaleway_dkim_record_name_value.webp" alt="" />
10. Add an [MX record](/managed-services/transactional-email/concepts/#mx-record).
11. Check the box "I have added these DNS records to my DNS zone" to confirm.
12. Click **Verify your domain**. Your domain displays in the list of your domains.

<Message type="note">

>  The verification of your domain might take up to 48 hours.

Reviewer: ```suggestion
 The verification of your domain can take up to 48 hours.
```

</Message>

## How to delete your domain

> 1. Click **Transactional Email** in the Network section of the [console](https://console.scaleway.com/) side menu. The list of your domains displays.

Reviewer: ```suggestion
1. Click **Transactional Email** in the Managed Services section of the [console](https://console.scaleway.com/) side menu. The list of your domains displays.
```
